lib/matplotlib/text.py

```diff
@@ -768,7 +794,8 @@ def draw(self, renderer):
             gc.set_url(textobj._url)
             textobj._set_gc_clip(gc)
 
-            angle = textobj.get_rotation()
+            angle = (textobj.get_rotation() + (180.0*transangle/np.pi))%360.0
+
```

Comment:
@WeatherGod I'm not sure that you wanted to comment on this line specifically and not the PR as a whole?